internal/service/templateversion.go

```diff
@@ -90,6 +91,27 @@ func (h *ServiceHandler) ListTemplateVersions(ctx context.Context, request serve
 // (DELETE /api/v1/api/v1/fleets/{fleet}/templateVersions)
 func (h *ServiceHandler) DeleteTemplateVersions(ctx context.Context, request server.DeleteTemplateVersionsRequestObject) (server.DeleteTemplateVersionsResponseObject, error) {
 	orgId := store.NullOrgId
+	listParams := store.ListParams{Limit: 100}
```

Comment:
can we add a comment why limit is hardcoded? If 100 is an important number maybe can we add it as a const?